### PR TITLE
[CDAP-18005] Re-enable Wrangler E2E tests to work with connection management

### PR DESCRIPTION
**Pull request by njbriggs**
Jira: https://cdap.atlassian.net/browse/CDAP-18005

### CODE REVIEW

File: cypress/support/constants.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ const DEFAULT_GCP_SERVICEACCOUNT_PATH = Cypress.env('gcp_service_account_path');
 const DEFAULT_GCP_PROJECTID = Cypress.env('gcp_projectid');
 
 const DEFAULT_GCS_CONNECTION_NAME = `gcs_${getRandomArbitrary()}`;
-const DEFAULT_GCS_FOLDER = 'cdap-gcp-ui-test';
+const DEFAULT_GCS_FOLDER = '000cdap-gcp-ui-test';
```
Comment (seanfeld) [inline cypress/support/constants.ts:27]: Why 000?

**Review by seanfeld — COMMENTED** [review]
Looks good but this is big maybe it should get another pair of eyes?

File: cypress/support/constants.ts (hunk 1 above)
Comment (njbriggs) [reply cypress/support/constants.ts:27]: Oh let me add a comment. The test bucket has been pushed down by other buckets so it doesn't appear in the listing.

**Review by lahwran — APPROVED** [review]
Looks good to my eye, so I'll approve.

**Pull request merged**